index.html

```diff
@@ -0,0 +1,637 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="style.css">
+  <title>Bootstrap Project</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <script src="https://kit.fontawesome.com/668d372fa7.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <main>
+    <header>
+      <div class="desktop-navbar">
+        <nav class="first-nav">
+          <i class="fas fa-cloud-sun">27°null, MU > </i>
+          <img src="images/logo.png" alt="logo">
+          <div class="right-nav">
+            <a href="#">SIGN IN</a>
+            <button type="button" class="button" name="button">Subscribe ></button>
+          </div>
+
+        </nav>
+
+        <nav class="low-nav ">
+          <a href="#">U.S |</a>
+          <a href="#">World |</a>
+          <a href="#">Tech & Science |</a>
+          <a href="#">Culture |</a>
+          <a href="#">Newsgeek |</a>
+          <a href="#">Sports |</a>
+          <a href="#">Health |</a>
+          <a href="#">The Debate |</a>
+          <a href="#">Weather </a>
+
+          <span>Search <i class="fas fa-search"></i></span>
+
+        </nav>
+
+      </div>
+      <div class="mobile-navbar">
+        <nav class="navbar  mobile-nav">
+          <div class="container-fluid">
+
+            <div class="navbar-header">
+              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+                <span class="sr-only">Toggle navigation</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+              </button>
+              <a class="navbar-brand" href="#">Newsweek</a>
+            </div>
+
+
+            <div class="collapse navbar-collapse nav-i" id="bs-example-navbar-collapse-1">
+              <ul class="nav navbar-nav">
+                <li class="active"><a href="#">SIGN IN <span class="sr-only">(current)</span></a></li>
+                <li><a href="#">SUBSCRIBE</a></li>
+              </ul>
+              <i class="fas fa-bars fa-lg"></i>
+            </div>
+          </div>
+        </nav>
```

Comment:
- [ ] You shouldn't create two red navbars. The one you have on top should be responsive and show/hide elements depending on the screen size. You can achieve that using the examples of bootstrap navbars.